lib/Sema/CSApply.cpp

```diff
@@ -575,7 +575,7 @@ namespace {
                     TypeExpr::createImplicitHack(loc.getBaseNameLoc(), baseTy,
                                                  ctx);
                   refExpr = new (ctx) MemberRefExpr(base, SourceLoc(), witness,
-                                                    loc, /*Implicit=*/true);
+                                                    loc, /*Implicit=*/false);
```

Comment:
I'm not sure if that's valid in general to have non-implicit expression without a valid source loc? I think last time I tried to do similar thing it tripped verifier...